### ISSUE TITLE
Revert inclusion of source files in Gradle's binary jars

### ISSUE THREAD

**Issue by cbeams** [gradle/gradle-script-kotlin#55]
Blocked on #43 and #44. When resolved, we will be able to upgrade to ij-kt-plugin version dev-467 or better and take advantage of the new source attachment features there. We can then revert the workaround inclusion of source files in Gradle's binary jars.

**Comment by cbeams** [comment]
I have confirmed that the latest custom gsk distros (`20160620*`) created after gradle/gradle@7e97cee
*do not* contain source files in `gradle-*` module jars following, whereas earlier gsk distros, e.g. `20160617*` *do* contain source files.

I'm therefore closing this issue, as this change will carry over to Gradle 3.0 M2 as soon as `gradle/gradle:gradle-script-kotlin` is merged into it.
